arbeitsplan/templates/home.html

```diff
@@ -12,191 +12,243 @@
 
 {% if msg %}
 <div class="alert alert-{{msgclass}}">
-{{ msg }}
+  {{ msg }}
 </div>
 <p>
-{% endif %}
+  {% endif %}
+
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'custom_css/svpb.css'%}" />
 
 <p>
-Dir stehen folgende Funktionen zur Verfügung: 
+  Dir stehen folgende Funktionen zur Verfügung:
 <p>
-
-<div class="row row-cols-1 row-cols-lg-2 g-4">
-  
-  <div class="col">        
-    <div class="card">
-      <div class="card-header text-bg-secondary text-bg-secondary text-center">
-        <i class="fa-solid fa-list-ul fa-lg fa-fw"></i>&nbsp;Aufgaben
-      </div>
-      <ul class="list-group list-group-flush">
-        <li class="list-group-item"><a href="/arbeitsplan/aufgaben/">Welche Aufgaben gibt es?</a></li>
-      </ul>
+<div class="row g-4 py-5 row-cols-1 row-cols-lg-4">
+  <div class="col d-flex align-items-start">
+    <div
+      class="icon-square text-body-emphasis bg-body-secondary d-inline-flex align-items-center justify-content-center fs-4 flex-shrink-0 me-3">
+      <i class="bi fa-solid fa-list-ul fa-lg" width="1em" height="1em"></i>
+    </div>
+    <div>
+      <a href="/arbeitsplan/aufgaben/" class="icon-link">Aufgaben anzeigen</a>
```

Comment:
Is `icon-link` needed here? If I understand https://getbootstrap.com/docs/5.3/helpers/icon-link/ correctly, this should be used to style an icon inside the link tag, but here there is none.